part-spec/sensor.json

```diff
@@ -0,0 +1,240 @@
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "sensor",
+    "oneOf": [
+        {
+            "title": "accelerometer",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "accelerometerType": {
+                    "description": "Type of accelerometer",
+                    "examples": [
+                        "capacitive MEMS",
+                        "piezoresistive",
+                        "piezoelectric"
+                    ],
+                    "type": "string"
+                },
+                "accelerationRanges": {
+                    "description": "range of force that accelerometer can measure",
+                    "examples": [
+                        "2",
+                        "4",
+                        "8",
+                        "16"
+                    ],
+                    "comment": "units of g",
+                    "type": "number"
+                },
+                "accelerationSensitivity": {
+                    "description": "smallest change in force the accelerometer is able to capture (typical). Corresponds to levels in acceleration ranges",
+                    "examples": [
+                        "16000",
+                        "8000",
+                        "4000",
+                        "2000"
+                    ],
+                    "comment": "units of LSB/g; conditions can include acceleration ranges and output resolution",
+                    "type": "number",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "accelerationSensitivityOverTemperature": {
+                    "description": "accelerometer sensitivity change over temperature",
+                    "comment": "units of percentage per degree celsius (%/Celcius)",
+                    "$ref": "definitions.json#/value"
+                },
+                "axis": {
+                    "description": "number of axes of acceleration measurement",
+                    "type": "number"
+                },
+                "zerogOffset": {
+                    "description": "output of the accelerometer when no acceleration is applied",
+                    "comment": "units of g-force (g)",
+                    "$ref": "definitions.json#/value"
+                },
+                "zerogOffsetOverTemperature": {
+                    "description": "accelerometer zero-g offset change over temperature",
+                    "comment": "units of milli g-force per degree celsius (mg/Celcius)",
+                    "$ref": "definitions.json#/value"
```

Comment:
Looks like this didn't get picked up, I think we want:

"type: "number"

NOT: "$ref": "definitions.json#/value"

Reply:
"$ref": "definitions.json#/value" should be ok here since we've redefined this as a range of numbers ( min, typ, max). This is for range values without defined units ( in the definitions file).